forms-mit/ask.html

```diff
@@ -55,7 +55,7 @@
       <!--IF THE LIST BELOW IS REORDERED, ADDE TO OR DELETED FROM, THERE IS A SCRIPT ABOVE THAT MUST BE CHANGED! Search for "document.ask.topic". This script determines which .txt file the form gets sent to, based on which topic is selected.-->
```

Comment:
I notice this says a script needs to be changed if this list is changed, but I can't find the script... Can you see if there is something still exists that needs to align with these changes?